qemu/tests/power_nvram.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import re
+
+from virttest import error_context
+from virttest import virt_vm
+from virttest import qemu_monitor
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    nvram test on power:
+    1) Boot vm with different raw string for nvram
+    2) Qemu works as expected
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    vm = env.get_vm(params["main_vm"])
+    error_msg = params.get('nvram_expected_result')
+    nvram_sub_type = params.get("nvram_sub_type")
+    if nvram_sub_type != "normal":
+        try:
+            vm.create(params=params)
+        except virt_vm.VMCreateError as e:
+            output = e.output
+            error_context.context("Check the expected error message: %s"
+                                  % error_msg, logging.info)
+            if not re.search(error_msg, output):
+                test.fail("Can not get expected error message: %s from %s"
+                          % (error_msg, output))
+        except qemu_monitor.MonitorConnectError as detail:
```

Comment:
```suggestion
        except qemu_monitor.MonitorConnectError:
```

BTW, why pass `MonitorConnectError` directly?

Reply:
Updated it and the second test case took much time to pop the warnings but it already started connecting monitor. Do you have any better solution for this situation? Thanks.

Reply:
Could you try to increase the timeout when creating the VM?
```
vm.create(params=params, timeout=timeout)
```

Reply:
It doesn't work even if the timeout was set to.

Reply:
Okay, you can use `debugonly = yes`.
Besides, please add `only ppc64 ppc64le`

Reply:
Updated it and keep the original way as we discussed yesterday, thanks.